core/imageroot/var/lib/nethserver/cluster/actions/alter-backup-repository/50update

```diff
@@ -44,6 +45,19 @@ if rprovider != rdb.hget(f'cluster/backup_repository/{rid}', 'provider'):
     json.dump([{'field':'provider','parameter':'provider','value': rprovider,'error':'backup_provider_mismatch'}], fp=sys.stdout)
     sys.exit(3)
 
+# validate repository credentials
```

Comment:
Why the validation is coded inside the `50update`? Is it better to move inside a `10validate`?

Reply:
I was thinking about splitting the step, then decided to do everything here because

- the step is already implementing some validation checks, I just added another to avoid too much copying and refactor
- the update part is just a matter of adding some keys to Redis: it's a few line of code, and it's fast

Reply:
ok!